### PR TITLE
First parts of scoreset wizardification.

### PR DESCRIPTION
**Pull request by ashsny**
By @jstone-uw and @ashsny.

### CODE REVIEW

File: src/components/screens/ScoreSetCreator.vue [hunk 1]
```diff
@@ -0,0 +1,2021 @@
...
+              <div class="field">
+                <Message v-if="targetGenes?.length > 1" severity="info">
+                  When defining variants against multiple targets, uploaded variant coordinates should be fully
+                  qualified with respect to target names or target accessions.
+                </Message>
+              </div>
+              <div class="mavedb-wizard-step-controls-row">
+                <div class="flex justify-content-between mavedb-wizard-step-controls pt-4">
+                  <Button label="Back" severity="secondary" icon="pi pi-arrow-left" @click="showPreviousWizardStep" />
+                  <Button label="Next" :disabled="this.maxWizardStepValidated < activeWizardStep" icon="pi pi-arrow-right" iconPos="right" @click="showNextWizardStepIfValid(showNextWizardStep)" />
```
Comment (bencap) [inline src/components/screens/ScoreSetCreator.vue:705]: This step is automatically valid, but we should probably enforce that the user has added at least one target.

File: src/components/screens/ScoreSetCreator.vue [hunk 2]
```diff
@@ -0,0 +1,2021 @@
...
+          // - The score set is a superseding score set, and the superseded score set has been chosen.
+          // - Or the score set is neither, and its parent experiment has been chosen.
+          return !!(this.isMetaAnalysis && this.metaAnalyzesScoreSets.length > 0)
+              || !!(this.isSupersedingScoreSet && this.supersededScoreSet)
+              || !!(!this.isMetaAnalysis && !this.isSupersedingScoreSet && this.experiment)
+        }
+        case 1: {
+          return this.title && this.shortDescription && this.abstractText && this.methodText
+        }
+
```
Comment (bencap) [inline src/components/screens/ScoreSetCreator.vue:1137]: Per the comment above, we could minimally add a check on the length of the user added target genes list to ensure they've added at least one before advancing:
```suggestion
case 2: {
      return this.targetGenes.length > 0
}
```

File: src/components/screens/ScoreSetCreator.vue [hunk 3]
```diff
@@ -0,0 +1,2021 @@
...
+      this.existingTargetGene = null
+      this.geneName = null
+      this.geneNameAccessionSuggestions = []
+      this.geneNameDropdownValue =  null
+      this.fileCleared('targetGeneTargetSequenceSequenceFile')
+      this.referenceGenome = null
+      this.targetGene = emptyTargetGene()
+    },
+
+    addTarget: function () {
```
Comment (bencap) [inline src/components/screens/ScoreSetCreator.vue:1564]: We'll need to integrate https://github.com/VariantEffect/mavedb-ui/pull/194 here. I can handle that in the other PR once this one is merged in.

**Review by bencap — COMMENTED** [review]
This is much nicer than the previous experience, left a couple questions/comments I had while going through the code alongside creating a new score set with the wizard flow.

File: src/components/layout/Toolbar.vue [hunk 4]
```diff
@@ -18,6 +18,19 @@
           </div>
         </div>
       </template>
+      <template #item="{item, props, hasSubmenu}">
+        <router-link v-if="item.route" v-slot="{href, navigate}" :to="item.route" custom>
+          <a v-ripple v-bind="props.action" class="p-menuitem-link" :href="href" @click="navigate">
```
[the remark this answers is not in the document]
Comment (ashsny) [reply src/components/layout/Toolbar.vue:23]: removed

File: src/components/layout/Toolbar.vue [hunk 5]
```diff
@@ -18,6 +18,19 @@
...
+            <span v-if="item.icon" :class="['p-menuitem-icon', item.icon]" />
+            <span class="p-menuitem-text">{{ item.label }}</span>
+          </a>
+        </router-link>
+        <a v-else v-ripple class="p-menuitem-link" :href="item.url" :target="item.target" v-bind="props.action">
```
[the remark this answers is not in the document]
Comment (ashsny) [reply src/components/layout/Toolbar.vue:28]: removed

File: src/components/screens/ScoreSetCreator.vue (hunk 1 above)
Comment (ashsny) [reply src/components/screens/ScoreSetCreator.vue:705]: done by your suggested change

File: src/components/screens/ScoreSetCreator.vue [hunk 6]
```diff
@@ -0,0 +1,2021 @@
...
+                              <div>Identifier: {{ slotProps.option.identifier }}</div>
+                              <div>Database: {{ slotProps.option.dbName }}</div>
+                            </div>
+                          </template>
+                        </Multiselect>
+                        <label :for="$scopedId('input-primaryPublicationIdentifiers')">Primary publication</label>
+                      </span>
+                      <span v-if="validationErrors.primaryPublicationIdentifiers" class="mave-field-error">{{
+                        validationErrors.primaryPublicationIdentifiers }}</span>
+                      <Message v-if="experiment" severity="info">
```
[the remark this answers is not in the document]
Comment (ashsny) [reply src/components/screens/ScoreSetCreator.vue]: Moved to top.

**Pull request merged**